public/app/core/components/QueryOperationRow/QueryOperationRow.tsx

```diff
@@ -65,6 +72,7 @@ export const QueryOperationRow: React.FC<QueryOperationRowProps> = ({
             }}
             aria-label="Query operation row title"
           >
+            {draggable && <Icon name="draggabledots" size="lg" className={styles.dragIcon} />}
```

Comment:
Super tiny nit pick - I would add a title to the draggable icon to let users know that you are able to reorder transformations by dragging the row.